lib/ares/http.rb

```diff
@@ -1,13 +1,13 @@
 module Ares
   module Http
     def get(url, params = {})
-      response = HTTParty.get(url, query: params)
+      response = HTTParty.get(url, query: params, timeout: Ares.timeout)
```

Comment:
How? Please can you add there documentation with examples how to change default timeout?
See here:
http://inch-ci.org/github/ucetnictvi-on-line/ares.rb